.github/workflows/helm-chart-release.yml

```
@@ -59,12 +60,29 @@ jobs:
           GPG_KEYRING_BASE64: "${{ secrets.GPG_KEYRING_BASE64 }}" #Referring secrets of github above
           GPG_PASSPHRASE: "${{ secrets.GPG_PASSPHRASE }}"
 
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
```

I think this is 3.5.0?
https://github.com/docker/login-action/commit/74a5d142397b4f367a81961eba4e8cd7edddf772

But maybe using this hash pointing to the tag is better?
https://github.com/docker/login-action/commit/184bdaa0721073962dff0199f1fb9940f07167d1

```suggestion
        uses: docker/login-action@184bdaa0721073962dff0199f1fb9940f07167d1 # v3.5.0
```